site/diagrams/sequenceDiagrams/protocolAdapterDeviceAuthentication_Success.txt

```diff
@@ -1,19 +1,13 @@
-title Protocol Adapter Device Authentication (success)
+title Protocol Adapter Device Authentication (MQTT, success)
 
-Device->+Protocol Adapter: authenticate(type, auth-id, secrets)
-note over Device:
-Denotes the process of authentication
-(that differs for every protocol)
-end note
+Device->+MQTT Protocol Adapter: CONNECT(username, hashed-password)
```

Comment:
FMPOV this should be
`Device->+MQTT Protocol Adapter: CONNECT(username, password)` 
because the device does in fact send a clear text password (not the hash).

Reply:
+1 : was a mistake, you are right.